src/styles/blocks/title.scss

```diff
@@ -0,0 +1,16 @@
+.title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 32px;
+  letter-spacing: -1.5px;
+
+  @include onTablet {
+    font-size: 48px;
```

Comment:
Consider using a variable for font-size on tablet breakpoint to keep the code maintainable.